vagga.yaml

```diff
@@ -51,6 +51,18 @@ containers:
       /run: !Tmpfs
         subdirs:
           postgresql: { mode: 0o777 }
+  test:
+    setup:
+      - !Ubuntu xenial
+      - !PipConfig
+        dependencies: true
+      - !BuildDeps
```

Comment:
You don't need to declare runtime `!Install` dependencies as `!BuildDeps` as they are available during build as well